skietbaan_fe/src/components/Notifications.js

```diff
@@ -2,42 +2,48 @@ import React, { Component } from "react";
 import "../components/NotificationsStyle.css";
 import { getCookie } from "./cookie";
 import history from "./history";
-import { URL } from "../actions/types.js";
+import { LOCALURL } from "../actions/types.js";
 
 class notification extends Component {
   constructor(props) {
     super(props);
     this.state = {
       array: [],
-      notificationMsg: "",
       typeOfNotification: "",
       tokenValue: "",
-      isRead: false,
-      showDelete: false,
       deleteClicked: false,
-      viewClicked: false
+      deleted: true,
+      cancelClicked: false
     };
     this.onDelete = this.onDelete.bind(this);
+    this.markForDeletion = this.markForDeletion.bind(this);
   }
 
-  onDelete = key => {
-    let id = this.state.array[key].id;
-    let newArray = this.state.array;
-    newArray.splice(key, 1);
-    this.setState({ array: newArray });
+  onDelete = async () => {
+    const deleteNotification = async id => {
+      try {
+        await fetch(LOCALURL + `/api/Notification/${id}`, {
```

Comment:
Is LOCALURL not a localhost url? Also all cap constants' words should be separated by '_' like LOCAL_URL